mobile/cmp/input/Select.js

```diff
@@ -385,14 +433,51 @@ const cmp = hoistCmp.factory(
             rsProps.formatCreateLabel = model.createMessageFn;
         }
 
-        const factory = model.creatableMode ? reactCreatableSelect : reactSelect;
+        const factory = model.creatableMode ? reactCreatableSelect : reactSelect,
+            control = factory(rsProps);
+
+        if (model.fullscreen) {
+            return ReactDom.createPortal(
+                fullscreenWrapper({
+                    model,
+                    title: props.fullscreenTitle,
+                    item: box({
+                        item: control,
+                        className,
+                        ref
+                    })
+                }),
+                model.getOrCreateFullscreenPortalDiv()
+            );
+        } else {
+            return box({
+                item: control,
+                className,
+                ...layoutProps,
+                width: withDefault(width, null),
+                ref
+            });
+        }
+    }
+);
 
-        return box({
-            item: factory(rsProps),
-            className,
-            ...layoutProps,
-            width: withDefault(width, null),
-            ref
+const fullscreenWrapper = hoistCmp.factory(
+    ({model, title, children}) => {
+        return div({
+            className: 'xh-select__fullscreen-wrapper',
+            items: [
+                toolbar({
+                    className: 'xh-select__fullscreen-toolbar',
+                    items: [
+                        button({
+                            icon: Icon.chevronLeft(),
+                            onClick: () => model.setFullscreen(false)
+                        }),
+                        span(title)
+                    ]
+                }),
+                children
```

Comment:
realize there is only a single child, but this is confusing.  Might either spread with ...children, or use React.Children.only(children)